plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -1007,26 +867,26 @@ void chain_plugin::plugin_initialize(const variables_map& options) {
                   ilog( "Using genesis state provided in '${genesis}'", ("genesis", genesis_file.generic_string()));
                }
                wlog( "Starting up fresh blockchain with provided genesis state." );
```

Comment:
This is the path where there is no existing block log.  This is essentially the same as it was before, except that I removed being able to just specify genesis-timestamp with a default genesis, and added checking also for not having my->chain_id set (to cover the case where we have a truncated/reset block_log).
If that was wrong before, I will need your help figuring out how the logic should change here.

Reply:
Yes, it was wrong before. Particularly in the case where someone deleted their block log (but still kept the state) and tried to restart nodeos. That would be re-using the existing state (not initializing a fresh one) but the logs would claim otherwise.

Reply:
So should I just drop these log statements?

Reply:
assuming that I should just drop them